dv-export/python/main.py

```diff
@@ -0,0 +1,59 @@
+import shutil
+
+from dotenv import dotenv_values
+from visier.connector import VisierSession, make_auth
+from visier.api import DVExportApiClient
+from data_store import *
+from dv_export import DVExport, DataVersions
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s [%(levelname)s] %(name)s: %(message)s',
+    filename='app.log',
+    filemode='w',
+)
+console_handler = logging.StreamHandler()
+console_handler.setLevel(logging.INFO)
+console_handler.setFormatter(logging.Formatter('%(asctime)s [%(levelname)s] %(name)s: %(message)s'))
+logger = logging.getLogger()
+logger.addHandler(console_handler)
+
+env_variables = dotenv_values()
+
+auth_keys = ['VISIER_HOST', 'VISIER_USERNAME', 'VISIER_PASSWORD', 'VISIER_APIKEY', 'VISIER_CLIENT_ID', 'VISIER_VANITY']
+auth_variables = {auth_key: env_variables[auth_key] for auth_key in auth_keys}
+auth = make_auth(env_values=auth_variables)
+
+dv_export_variables = {
+    export_key: env_variables[export_key] for export_key in env_variables.keys() if export_key not in auth_keys
+}
+
+with VisierSession(auth) as s:
+    dv_export_client = DVExportApiClient(s, raise_on_error=True)
+
+    data_version_number_raw = dv_export_variables['DATA_VERSION_NUMBER']
```

Comment:
Consider moving to e.g. constants.py file so you can refer to symbolic constants in code as opposed to strings